### DISCUSSION TITLE
How to output the concentration gradient energy contribution from "CHInterface"

### DISCUSSION THREAD

**Discussion by wenpeng231024** [idaholab/moose#18447] [Q&A General] [answered]
Hi everyone,

I use "CHInterface" to calculate the concentration gradient energy. Does anyone know how to output this part gradient energy?

![image](https://user-images.githubusercontent.com/60105926/127012016-ba029a82-b940-4af6-a7a4-0cb01601deff.png)

https://mooseframework.inl.gov/source/kernels/CHInterface.html

Thanks,

Peng

**Comment by dschwen** [comment, marked as answer]
You can use the `TotalFreeEnergy` AuxKernel. Just set `F_name = 0` to ignore the bulk contribution.

**Reply by wenpeng231024** [reply]
Thanks for your reply. I will try this.
